app/services/geetest_verifier.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+require 'digest'
+require 'net/http'
+
+class GeetestVerifier
```

Comment:
We should look for the opportunity to merge both types of captcha in one service file